### PR TITLE
Optimize the activation method of fport2

### PR DESCRIPTION
**Pull request by WismyYao**

### CODE REVIEW

File: radio/src/gui/480x272/model_receiver_options.cpp [hunk 1]
```diff
@@ -171,21 +179,19 @@ bool menuModelReceiverOptions(event_t event)
...
-          }
-          break;
-
-        case ITEM_RECEIVER_SETTINGS_FPORT2:
-          lcdDrawText(MENUS_MARGIN_LEFT, y, "F.Port2");
-          reusableBuffer.hardwareAndSettings.receiverSettings.fport2 = editCheckBox(reusableBuffer.hardwareAndSettings.receiverSettings.fport2, RECEIVER_OPTIONS_2ND_COLUMN, y, attr, event);
-          if (attr && checkIncDec_Ret) {
+        case ITEM_RECEIVER_SETTINGS_FPORT:
+        {
+          lcdDrawText(0, y, "PORT");
```
Comment (3djc) [inline radio/src/gui/480x272/model_receiver_options.cpp]: You seem to be using STR_PROTOCOL for b&w, why hardcoding a (different) string here ?

Comment (WismyYao) [reply radio/src/gui/480x272/model_receiver_options.cpp]: Sorry for my mistake

File: radio/src/gui/common/stdlcd/model_receiver_options.cpp [hunk 2]
```diff
@@ -82,8 +91,7 @@ void menuModelReceiverOptions(event_t event)
     0, // PWM rate
     isModuleR9MAccess(g_moduleIdx) && receiverVariant == PXX2_VARIANT_EU && reusableBuffer.hardwareAndSettings.moduleSettings.txPower > 14 /*25mW*/ ? READONLY_ROW : (uint8_t)0, // Telemetry
     IF_RECEIVER_CAPABILITY(RECEIVER_CAPABILITY_TELEMETRY_25MW, 0),
-    IF_RECEIVER_CAPABILITY(RECEIVER_CAPABILITY_FPORT, 0),
-    IF_RECEIVER_CAPABILITY(RECEIVER_CAPABILITY_FPORT2, 0),
+    IF_RECEIVER_CAPABILITY(RECEIVER_CAPABILITY_FPORT, 0) || IF_RECEIVER_CAPABILITY(RECEIVER_CAPABILITY_FPORT2, 0),
```
Comment (bsongis) [inline radio/src/gui/common/stdlcd/model_receiver_options.cpp]: very wrong!

File: radio/src/translations/en.h.txt [hunk 3]
```diff
@@ -1225,6 +1225,7 @@
 #define TR_JITTER_FILTER               "ADC filter"
 #define TR_RTC_CHECK                   TR("Check RTC", "Check RTC voltage")
 #define TR_AUTH_FAILURE                "Auth-failure"
+#define TR_PROTOCOL                    "Protocol"
```
Comment (bsongis) [inline radio/src/translations/en.h.txt]: string defined 2 times, no?

**Comment by bsongis** [comment]
Please confirm once tests are OK so that I can merge it, thanks!

**Pull request merged**